feg/cloud/go/services/feg_relay/gw_to_feg_relay/tests/relay_test.go

```diff
@@ -150,6 +286,37 @@ func TestNHRouting(t *testing.T) {
 		t.Fatal("Neutral Host Routed S6a Proxy Call timed out")
 	}
 
+	// Test UpdateSession routing
+	sessProxyClient := lte_protos.NewCentralSessionControllerClient(conn)
+	toutctx, cancel = context.WithTimeout(ctx, 5*time.Second)
+	updateSessReq := &lte_protos.UpdateSessionRequest{
+		Updates: []*lte_protos.CreditUsageUpdate{
+			createCreditUsageRequest(nhImsi, 1, 1, lte_protos.CreditUsage_QUOTA_EXHAUSTED, lte_protos.RATType_TGPP_LTE),
+			createCreditUsageRequest(nhImsi2, 2, 2, lte_protos.CreditUsage_TERMINATED, lte_protos.RATType_TGPP_NR),
+		},
+		UsageMonitors: []*lte_protos.UsageMonitoringUpdateRequest{
+			createUsageMonitoringRequest(nhImsi, "mkey", 1, lte_protos.MonitoringLevel_SESSION_LEVEL, lte_protos.RATType_TGPP_LTE),
+			createUsageMonitoringRequest(nhImsi2, "mkey2", 2, lte_protos.MonitoringLevel_PCC_RULE_LEVEL, lte_protos.RATType_TGPP_NR),
+		},
```

Comment:
can you add one monitoring request more and. This way the test will cover more cases with 2 updates and 3 monitor request

Reply:
@uri200 , Done. Please check